Add admin API to send trace notifications to registered (#7128)
Minio-io/minio: pkg/madmin/README.md

    log.Println("Profiling data successfully downloaded.")
```

<a name="ListenTrace"></a>
### ListenTrace(allTrace bool,doneCh <-chan struct{}) <-chan TraceInfo

> get http trace from minio server

Reviewer: We should explain a bit more here, that how it is that trace comes from all servers in a cluster etc. Start the sentence with a capitalization since this is a document.

> Ask all nodes in minio cluster to send trace of HTTP requests

Reviewer: ```suggestion
Enable HTTP request tracing on all nodes in a MinIO cluster
```
